### DISCUSSION TITLE
Feature request: URL bar for Zen Sidebar

### DISCUSSION THREAD

**Discussion by georg-stone** [zen-browser/desktop#2856] [Ideas]
Zen Sidebar is an interesting concept but could use a URL bar for easier navigation without creating new base URLs.

**Comment by menganatha** [comment]
So much this, it would be incredibly more convenient. E.g you could quickly google something, visit some site, then press home button after you are done, without having to switch to a specialized google only panel (which could already contain some data that you'd like to not overwrite) and then again switch back from it.
Floorp 12 is going to support this.
